pkg/cvo/cvo.go

```diff
@@ -678,15 +678,25 @@ func (optr *Operator) sync(ctx context.Context, key string) error {
 
 	// identify the desired next version
 	desired, ok := findUpdateFromConfig(config, optr.getArchitecture())
-	if ok {
-		klog.V(2).Infof("Desired version from spec is %#v", desired)
+	stillInitializing := optr.configSync.StillInitializingFunc()()
+	if ok && !stillInitializing {
+		klog.V(2).Infof("Desired version from spec is %#v after initialization", desired)
 	} else {
+		userDesired := desired
```

Comment:
Maybe call this `pendingDesired`? Something that we still need to evaluate?

Reply:
Done